### DISCUSSION TITLE
auth_groups_users work with db

### DISCUSSION THREAD

**Discussion by pippuccio76** [codeigniter4/shield#1057] [Q&A]
HI , there is a away to change users group working with db ?

**Comment by datamweb** [comment]
Hi,
See https://shield.codeigniter.com/references/authorization/#managing-user-groups

**Reply by pippuccio76** [reply]
In your response there isn't db table to select group to assign to user or am I doing it wrong?

**Reply by datamweb** [reply]
Do you mean instead of using the following config file:

https://github.com/codeigniter4/shield/blob/933da424a73c1569baf2d33f21f8460a5f923419/src/Config/AuthGroups.php#L43-L64

Store data in a database table?

**Reply by pippuccio76** [reply]
Yes

**Reply by datamweb** [reply]
We do not have a separate table for this. But instead we use **CodeIgniter Settings** library. By default, it is possible to save the new group in the database using this library.
Read [here](https://settings.codeigniter.com/) for more details.
